README.md

```diff
@@ -29,7 +29,23 @@
 
 > Tweet me <a target="_blank" rel="noopener" href="https://vas.cx/twitter">@vsnthdev</a>, I would love to know your opinion/experience on this project 😍
 
-<!-- ## 📘 Usage -->
+## 📘 Usage 
+
+If you want to *@mention* someone, or use a *link*, or even use a *#hashtag* in your tweets without notifying anyone, then **twrm** is here to help you out!  
+
+*Here's how it works : 👇*
+1. Go to  [twrm.io](https://twrm.io/) and use the search box to @mention the twitter handle of the person (from whom you don't want to get notified!)
```

Comment:
Replace "search box" with "text box" since we're not searching anything with that box, we don't call it a search box.